tests/system/src/test/java/test/robot/javafx/scene/dialog/JDK8193502Test.java

```diff
@@ -0,0 +1,131 @@
+package test.robot.javafx.scene.dialog;
```

Comment:
This needs a copyright header (see any of the other test files for an example). The year should be just `2019, `

Reply:
done.